sorc/link_workflow.sh

```diff
@@ -131,10 +131,6 @@ fi
 #---------------------------------------
 #--add files from external repositories
 #---------------------------------------
-#--copy/link NoahMp table form ccpp-physics repository
-cd "${HOMEgfs}/parm/ufs/land" || exit 1
-  ${LINK_OR_COPY} "${HOMEgfs}/sorc/ufs_model.fd/FV3/ccpp/physics/physics/noahmptable.tbl" .
```

Comment:
I think you misunderstood Rahul. This part was correct and is required by NCO. All the job script calls should look in this location, not down into `ufs_model.fd`. We just don't need the file in the repo since it will be linked in here (it should be added to the `.gitignore`)